packages/bits/src/animations/expand.ts

```diff
@@ -13,3 +15,21 @@ export const expand = trigger("expandedState", [
         animate("350ms ease-in-out"),
     ]),
 ]);
+
+    /**
+     * This animation is using the special selectors that ngIf and ngFor use on newly inserted or removed content.
+     */
+export const expandV2 = trigger("expandContent", [
+    transition(":enter", [
+        style({ height: 0 }),
+        animate("350ms ease-in", style({ height: "*" })),
+        query("@*",
+            stagger(300, [
```

Comment:
Does this need to be >= to 350 (instead of 300) to meet or exceed the 350ms animation time? Is that possibly why there is a jump when expanding the nested ones?

Reply:
The stagger is just a delay for each element to enter the animation. We dont want the stagger to be the same as the animation time because then it wont start the animation until the parent finishes it's animation. Does that make sense?